### ISSUE TITLE
Support Multi-task learning

### ISSUE THREAD

**Issue by Oufattole** [Oufattole/meds-torch#21]
We currently support single task supervised training, we should support multiple tasks. Users would input a list of task parquet files (with patient_id, end-point timestamp, and labels) and we should join these dataframes and then integrate them into the pytorch_dataset class [here](https://github.com/Oufattole/meds-torch/blob/973d88fcae8228433d1a5889bbd62b294c4666ea/src/meds_torch/pytorch_dataset.py#L411).
